src/openforms/scss/components/builder/_columns.scss

```diff
@@ -38,8 +42,17 @@ $bootstrap-ratio-desktop: math.div(12, $grid-columns-desktop);
   }
 
   .column {
-    $col-sizes: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11;
+    display: flex;
+    flex-direction: column;
+
+    padding-left: 3px;
+    padding-right: 3px;
 
+    & > * {
+      flex-grow: 1;
+    }
+
+    $col-sizes: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11;
```

Comment:
Since you added `{label: '12/12 (100%)', value: '12'},` to the column component configuration, do you need to also add the `12` to the column sizes?

Reply:
ah yeah, did it in the SDK but forgot to do so here :facepalm: